packages/world/foundry.toml

```diff
@@ -1,6 +1,6 @@
 [profile.default]
 solc = "0.8.21"
-ffi = false
+ffi = true
```

Comment:
This setting is relevant only for testing. [As the documentation states](https://book.getfoundry.sh/reference/config/testing#ffi), enabling this allows tests to execute arbitrary programs. However, since we are already running TypeScript tests, this setting should not have any impact.